### PR TITLE
add temp help component screen

### PR DESCRIPTION
**Pull request by kirayoon**
googoo gaga

### CODE REVIEW

**Review by Mrrm2 — APPROVED** [review]
lgtm much better than the last one

File: client/components/profile/ProfileStack.jsx [hunk 1]
```diff
@@ -16,6 +17,7 @@ export default function ProfileStack() {
       <Stack.Screen name="ProfileScreen" component={ProfileScreen} />
       <Stack.Screen name="EditInfo" component={EditInfo} />
       <Stack.Screen name="UserSettings" component={UserSettings} />
+      <Stack.Screen name="Help" component={Help} />
```
Comment (kirayoon) [inline client/components/profile/ProfileStack.jsx:20]: smart choice.

File: client/components/profile/ProfileStack.jsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { createStackNavigator } from "@react-navigation/stack";
 import ProfileScreen from "./ProfileScreen";
 import EditInfo from "./EditInfo";
 import UserSettings from "./UserSettings";
+import Help from "./Help";
```
Comment (kirayoon) [inline client/components/profile/ProfileStack.jsx:7]: yes.

File: client/components/profile/ProfileScreen.jsx [hunk 3]
```diff
@@ -77,7 +77,7 @@ export default function ProfileScreen({ navigation }) {
           />
           <ProfileScreenCard
             title="Help"
-            onPress={() => console.log("Help")}
+            onPress={() => navigation.navigate("Help")}
```
Comment (kirayoon) [inline client/components/profile/ProfileScreen.jsx:80]: this is good.

**Review by kirayoon — COMMENTED** [review]
loving your work so far. keep it up man.

File: client/components/profile/ProfileScreen.jsx (hunk 3 above)
Comment (Mrrm2) [reply client/components/profile/ProfileScreen.jsx:80]: agree

**Pull request merged**